proto/logbackuppb.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package logbackup;
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+import "errorpb.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+option java_package = "org.tikv.kvproto";
+
+// The minimal information for identify a region.
+message RegionIdentity {
+    uint64 id = 1;
+    uint64 epoch_version = 2;
+    // We omitted epoch_conf_version because config change won't make range change.
+}
+
+// The last flush ts with region information.
+message RegionCheckpoint {
+    errorpb.Error err = 1;
+    RegionIdentity region = 2;
+    uint64 checkpoint = 3;
+}
+
+message GetLastFlushTSOfRegionRequest {
+    repeated RegionIdentity regions = 1;
```

Comment:
shall we add limit here 🤔

Reply:
One `RegionIdentity` has 16 bytes.
if we have 100k region in a store seems 1.5M size in one request is fine.